Sources/Documentation/XCStringsToolPlugin.docc/Integrating XCStrings Tool into a Swift Package Target.md

```diff
@@ -35,7 +35,7 @@ let package = Package(
     ],
     dependencies: [
         // 1. Add the xcstrings-tool Package dependency
-        .package(url: "https://github.com/liamnichols/xcstrings-tool-package.git", from: "0.1.0")
+        .package(url: "https://github.com/liamnichols/xcstrings-tool.git", from: "0.1.0")
```

Comment:
Whoops… thanks for spotting this! 

In fact, the correct url is supposed to be the plugin repository that uses a precompiled version of the cli:

```suggestion
        .package(url: "https://github.com/liamnichols/xcstrings-tool-plugin.git", from: "0.1.0")
```

🙏